### PR TITLE
Add Nullstone to ecosystem

### PR DESCRIPTION
**Pull request by BSick7**

### CODE REVIEW

File: content/ecosystem/paas-devops-platforms/nullstone.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
++++
+title="Nullstone"
+aliases="/frameworks/nullstone/"
```
Comment (christophcrichter) [inline content/ecosystem/paas-devops-platforms/nullstone.md:3]: Can you please remove this line? It's not needed in the header.

**Review by christophcrichter — CHANGES_REQUESTED** [review]
Overall, thanks for contributing. I left one comment.

File: content/ecosystem/paas-devops-platforms/nullstone.md (hunk 1 above)
Comment (BSick7) [reply content/ecosystem/paas-devops-platforms/nullstone.md:3]: Ah, I copied from the format of others. I will drop this line.

**Review by christophcrichter — APPROVED** [review]
LGTM; thanks for the adjustment

**Pull request merged**